plugins-builtin/src/main/scala/za/co/absa/enceladus/plugins/builtin/kafka/ControlInfoAvroSerializer.scala

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
For consideration: I would add a `controlMetricsPlugin` package before `kafka`. And maybe not use `builtin`.

Reply:
I agree on the order of elements of the package tree. But I want to have `builtin` as the part of the same so that when someone uses a plugin in the configuration it would be obvious if that plugin is an internal (built-in) or external one. For instance, if `ClassNotFound` exception is thrown for an external plugin a JAR might be missing. If the plugin is a built-in one the issue is differet, maybe a bug in fat JAR creation.

Reply:
Yeah, good point about the `builtin` 👍